arch/src/main/java/dk/ufst/arch/Store.kt

```diff
@@ -64,6 +71,23 @@ open class GlobalStore<Value, Action, Environment>(
     }
 
     fun destroy() {}
```

Comment:
Is `destroy` still needed?

Reply:
no I just didn't want to fix stuff unrelated to the subject of the PR, but guess I did anyway by changing getSubscriberCount haha so why not :)